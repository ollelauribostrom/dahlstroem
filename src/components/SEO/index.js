import SEO from './SEO';
import './SEO.css';

export default SEO;
