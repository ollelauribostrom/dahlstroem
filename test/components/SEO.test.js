import React from 'react';
import { shallow } from 'enzyme';

import SEO from '../../src/components/SEO';

describe('Tests for SEO', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SEO title="Test" />);
    expect(wrapper).toMatchSnapshot();
  });
});
