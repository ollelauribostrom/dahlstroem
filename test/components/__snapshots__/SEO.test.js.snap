// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for SEO renders correctly 1`] = `
<mockConstructor
  render={[Function]}
/>
`;
